### ISSUE TITLE
New release?

### ISSUE THREAD

**Issue by Ruin0x11** [discordrb/discordrb#753]
I want to use `allowed_mentions` with webhooks to prevent the use of `@everyone`, but I have to use the version from latest master to have that feature.

It would be better in my opinion if I could use an official release.

**Comment by gracz21** [comment]
+1, the latest gem release on the RubyGems is from late 2020. Almost 3 yrs later and there were no release. In the mean time the gem had a lot of changes (e.g. change of API from V6 to V9 which took me by surprise)

**Comment by Daniel-Worrall** [comment]
This project is now being tracked at https://github.com/shardlab/discordrb and has just released 3.5.0

**Comment by gracz21** [comment]
Yes, yes, I removed my comment here, posted a comment on the proper new repo place & the new version is released
